.gitignore

```diff
@@ -1,2 +1,3 @@
 lib
 umd
+node_modules
```

Comment:
This actually didn't get taken out, but I think it's fine to keep - it's part of standard `.gitignore` for Node per e.g. https://github.com/github/gitignore/blob/master/Node.gitignore. Most other JS projects ignore `node_modules`, per e.g. https://github.com/rackt/redux/blob/v3.0.4/.gitignore#L3.

Typically I think the user-global `.gitignore` is better suited for editor configuration rather than anything even slightly project-specific.